src/emcpy/plots/create_plots.py

```diff
@@ -497,6 +498,22 @@ def _map_contour(self, plotobj, ax):
         if plotobj.colorbar:
             self.cs = cs
 
+    def _map_filled_contour(self, plotobj, ax):
+
+        skipvars = ['plottype', 'longitude', 'latitude', 'data',
```

Comment:
Just a question on the choice of `skipvars` as a name.  Does it have a specific meaning in this context?  I'm not super familiar with emcpy so I might be missing something.

Reply:
If I recall correctly, it's what keys in the dictionary to not pass to the underlying matplotlib plotting routine

Reply:
Yes, it means exactly what you think. In map_plot.py, there are variables that are added to python object so we need to skip certain ones that are not related to the kargs that are for plot related things themselves.